*** IPTVApiModel.js     Sun, 11 Sep 2022 19:47:36 +0900
--- IPTVApiModel.js     Sun, 11 Sep 2022 19:47:36 +0900
***************
*** 66,71 ****
--- 66,72 ----
                  if (channel.hasLogoData) {
                      logo = `tvg-logo="${isSecure ? 'https' : 'http'}://${base}/api/channels/${channel.id}/logo"`;
                  }
+                 str += `#KODIPROP:mimetype=video/mp2t\n`;
                  str += `#EXTINF:-1 tvg-id="${channel.id}" ${logo} group-title="${channel.channelType}",${channelName}　\n`;
                  str += `${isSecure ? 'https' : 'http'}://${base}/api/streams/live/${channel.id}/m2ts?mode=${mode}\n`;
              }
